### DISCUSSION TITLE
Can a brim be printed with two layers?

### DISCUSSION THREAD

**Discussion by paulnaude** [OrcaSlicer/OrcaSlicer#5737] [Ideas]
I normally don't even use a brim on my parts but now encountered a part that even tears off from the brim due to the amount of warping (its a case like shape with a 2mm thick base and walls). The brim is wide enough to stay stuck to the heated bed but the warping is so severe that the part separates itself from the brim. I'm hoping that it could be a simple gcode mod to print an extra layer for the brim to make it just that much stronger to keep the part on the bed.

**Comment by winbard** [comment]
Hi,

that is what i want know too.
The skirt can chose the layer numbers. By the brim not.
It is possible that can be changed in the future?

**Comment by HAndreasH** [comment]
If this is not possible, you could change something in the design to have a better (larger) support surface. Perhaps add the brim or mouse ears at the corners at the height of 2 layers already in the design.
I sometimes do this too, but then you have to do more post-processing on the finished model.

**Comment by Mqrius** [comment]
You could try making the first layer double thick. There's a global setting for First Layer thickness.

**Reply by otede** [reply]
You can also easily manually add brim-like features in Orca
Slant3D - Stop Using Brims, Do This Instead - YouTube — https://www.youtube.com/watch?v=MCcFMDv_4eo

**Comment by jameszy780** [comment]
Use the Raft settings on the Support Tab...

**Reply by otede** [reply]
Is raft supposed to have morze holding power? I'm asking because recently I wasnt able to print a 7x30 mm part that was 60 mm tall without it detaching from the BL A1 textured plate.

**Reply by jameszy780** [reply]
It may. But the question was if a brim could be 2 layers... A 2 layer brim is called a raft :)

If your having adhesion issues, I'd scrub your plate with Dawn power wash and hot water.. blot dry and stick it back on the bed and warm it up.. still a problem? Flip it over user the other side or get a new plate.

**Reply by Reexto** [reply]
Use the Raft is good for adhesion but i used it in the beta version 2.0.0 i didnt liked that my print did not removed clean the raft was still connected with the print, you know better way to get it clean removed? which z-distance to the model u used?

**Comment by HAndreasH** [comment]
A better / different print bed could be a solution, or an enclosure for the printer.
Or the material is simply not suitable for this particular case because of its tendency to warp.
Instead of a raft, the idea from this video might be helpful: https://www.youtube.com/watch?v=gbtlL8mTqJI&t=639s

**Comment by LJKraus** [comment]
Reduce or eliminate the Brim-object gap.

If that is not enough, use your design program to add a brim of whatever thickness you wish.
